### PR TITLE
Updated info in parallel to the template #684

### LINKED ISSUE #684: Missing info in parallel to the template
Documentation page: get-started/sql-pipeline

comparing this guide with https://github.com/ploomber/projects/blob/master/templates/spec-api-sql/README.ipynb, the notebook mentions how to create the sample data but it's missing in the docs. Also, in notebook it's mentioned as SQLScript: config.get_client but in the doc it's SQLScript: clients.get_client. Minor changes that can help users understand the flow.

### PR DESCRIPTION
**Pull request by qixuan27**
## Describe your changes
Missing info in parallel to the template #684

## Issue ticket number and link
Closes #684 https://github.com/ploomber/ploomber/issues/684

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests (when necessary).
- [x] I have added the right documentation (when needed). Product update? If yes, write one line about this update.

### CODE REVIEW

File: doc/get-started/sql-pipeline.rst [hunk 1]
```diff
@@ -26,9 +26,40 @@ Or run it locally:
 .. code-block:: console
 
     ploomber examples --name templates/spec-api-sql
+  
+  
+You can also refer to https://github.com/ploomber/projects/blob/master/templates/spec-api-sql/README.ipynb for more information on using SQL scripts to manipulate data in a database, dump a table, and plot it with Python.
```
Comment (edublancas) [inline doc/get-started/sql-pipeline.rst:31]: please add the readme link using .rst link format:

```rst
`Link text <link URL>`_
```

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

File: doc/get-started/sql-pipeline.rst [hunk 2]
```diff
@@ -26,9 +26,40 @@ Or run it locally:
...
+    # create sample data
+    cd setup
+    bash setup.sh
+    # move back to the original spec-api-sql folder
+    cd ..
+
+with expected output as:
 
-Connecting to databases
+.. code-block:: txt
```
Comment (edublancas) [inline doc/get-started/sql-pipeline.rst:49]: no need to put the sample output here

File: doc/get-started/sql-pipeline.rst [hunk 3]
```diff
@@ -26,9 +26,40 @@ Or run it locally:
...
+You can also refer to https://github.com/ploomber/projects/blob/master/templates/spec-api-sql/README.ipynb for more information on using SQL scripts to manipulate data in a database, dump a table, and plot it with Python.
 
+Creating Sample Data
+-----------------------
+
+To create sample data, you can run the following code:
+
+.. code-block:: bash
+
+    %%bash
```
Comment (edublancas) [inline doc/get-started/sql-pipeline.rst:40]: please remove the %%bash - this only works in jupyter/ipython

**Review by edublancas — CHANGES_REQUESTED** [review]

**Comment by idomic** [comment]
@qixuan27 Did you run the PR locally and checked it looks like you're expecting?
Also a comment for next PRs try to be specific with your commit messages, looks like all of your commits are "Update sql-pipeline.rst" which is quite confusing and hard to revert.
A few more things you can try to improve is auto linting, and self code review before committing/pushing like we discussed.

**Comment by qixuan27** [comment]
Yes, I did. The current flow works well for ploomber/projects.
Sorry for the inconvenience. I did not comment for every update so the GitHub automatically commented on my commits like this. I will comment on each commit more precisely and use auto-linting before submitting pr.

**Comment by idomic** [comment]
Sounds good, overall good job!

**Pull request merged**